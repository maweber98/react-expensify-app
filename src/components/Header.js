import React from 'react';
import { Link, NavLink } from 'react-router-dom';

const Header = () => (
  <header className="header">
    <div className="content-container">
      <div className="header__content">
        <Link className="header__title" to="/">
          <h1>Expensify</h1>
        </Link>
        <NavLink to="/create" activeClassName="is-active">Create Expense</NavLink>
      </div>
    </div>
  </header>
);

export default Header;
