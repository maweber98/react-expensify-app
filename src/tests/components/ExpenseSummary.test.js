import React from 'react';
import { shallow } from 'enzyme';
import { ExpenseSummary } from '../../components/ExpenseSummary';

test('should correctly render ExpenseSummary with 1 expense', () => {
    const wrapper = shallow(<ExpenseSummary expenseCount={1} expenseTotal={0.99}/>);
    expect(wrapper).toMatchSnapshot();
});

test('should correctly render ExpenseSummary with 2 expenses', () => {
    const wrapper = shallow(<ExpenseSummary expenseCount={2} expenseTotal={1.98}/>);
    expect(wrapper).toMatchSnapshot();
});