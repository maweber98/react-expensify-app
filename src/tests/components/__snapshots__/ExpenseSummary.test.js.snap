// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render ExpenseSummary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $0.99
  </h1>
</div>
`;

exports[`should correctly render ExpenseSummary with 2 expenses 1`] = `
<div>
  <h1>
    Viewing 
    2
     
    expenses
     totalling 
    $1.98
  </h1>
</div>
`;
